src/metrics/statsbeat/statsbeat.ts

```diff
@@ -2,39 +2,57 @@ import * as os from "os";
 
 import { Logger } from "../../shared/logging";
 import {
+    AttachStatsbeatProperties,
+    CommonStatsbeatProperties,
     NetworkStatsbeat,
+    NetworkStatsbeatProperties,
     StatsbeatAttach,
     StatsbeatCounter,
     StatsbeatFeature,
     StatsbeatFeatureType,
     StatsbeatInstrumentation,
     StatsbeatResourceProvider,
+    EU_CONNECTION_STRING,
+    EU_ENDPOINTS,
+    NON_EU_CONNECTION_STRING,
 } from "./types";
 import { ApplicationInsightsConfig, AzureVirtualMachine, ResourceManager } from "../../shared";
 import { Util } from "../../shared/util";
-import { MetricTelemetry, MetricPointTelemetry } from "../../declarations/contracts";
 import { KnownContextTagKeys } from "../../declarations/generated";
 import { IVirtualMachineInfo } from "../../shared/azureVirtualMachine";
+import { MeterProvider, PeriodicExportingMetricReader, PeriodicExportingMetricReaderOptions } from "@opentelemetry/sdk-metrics";
+import { AzureMonitorExporterOptions, AzureMonitorStatsbeatExporter } from "@azure/monitor-opentelemetry-exporter";
+import { BatchObservableResult, Meter, ObservableGauge, ObservableResult } from "@opentelemetry/api-metrics";
+import { ExportResult, ExportResultCode } from "@opentelemetry/core";
 
 const STATSBEAT_LANGUAGE = "node";
+const AZURE_MONITOR_STATSBEAT_FEATURES = "AZURE_MONITOR_STATSBEAT_FEATURES";
 
 export class Statsbeat {
-    private _connectionString =
-        "InstrumentationKey=c4a29126-a7cb-47e5-b348-11414998b11e;IngestionEndpoint=https://dc.services.visualstudio.com/";
+    private _commonProperties: CommonStatsbeatProperties;
+    private _networkProperties: NetworkStatsbeatProperties;
+    private _attachProperties: AttachStatsbeatProperties;
     private _collectionShortIntervalMs = 900000; // 15 minutes
-    private _collectionLongIntervalMs = 1440000; // 1 day
+    private _collectionLongIntervalMs = 86400000; // 1 day
     private _TAG = "Statsbeat";
     private _networkStatsbeatCollection: Array<NetworkStatsbeat>;
     private _resourceManager: ResourceManager;
-    private _handle: NodeJS.Timer | null;
-    private _longHandle: NodeJS.Timer | null;
     private _isEnabled: boolean;
     private _isInitialized: boolean;
     private _config: ApplicationInsightsConfig;
     private _statsbeatConfig: ApplicationInsightsConfig;
     private _isVM: boolean | undefined;
-    private _statbeatMetrics: Array<{ name: string; value: number; properties: unknown }>;
     private _azureVm: AzureVirtualMachine;
+    
+    private _networkStatsbeatMeter: Meter;
+    private _networkStatsbeatMeterProvider: MeterProvider;
+    private _networkAzureExporter: AzureMonitorStatsbeatExporter;
+    private _networkMetricReader: PeriodicExportingMetricReader;
+
+    private _longIntervalStatsbeatMeter: Meter;
```

Comment:
Long Interval Statbeat Metrics should be handled by the Exporter now right?, is this going to cause duplicated attach and feature metrics?

Reply:
The case I was considering when leaving it in was if a customer produces only logs, I don't see the azure-montior-exporter ever being instantiated. If this isn't possible or the likelihood of it is so low that it's not relevant it should absolutely be removed since you're correct that it would generate duplicate long-interval statsbeats for any customer that produces more than only logs.